benchexec/tablegenerator/react-table/src/components/Overview.js

```diff
@@ -160,6 +167,7 @@ export default class Overview extends React.Component {
     const deserializedFilters = this.filterUrlRetriever() || [];
     if (!deepEqual(this.lastFiltered, deserializedFilters)) {
       // we only want to kick off filtering when filters changed
+      this.addTypeToFilter(deserializedFilters);
```

Comment:
Won't this break the caching because `deepEqual` in the line above will never return `true` anymore?

Reply:
Yes, you're right. I've moved the function that adds the column type upwards, before the `deepEqual` check is made.